web/packages/teleport/src/Users/UserAddEdit/UserAddEdit.tsx

```diff
@@ -80,37 +84,46 @@ export function UserAddEdit(props: ReturnType<typeof useDialog>) {
           <DialogHeader>
             <DialogTitle>{isNew ? 'Create User' : 'Edit User'}</DialogTitle>
           </DialogHeader>
-          <DialogContent>
+          <DialogContent maxHeight={620} overflow={'auto'}>
             {attempt.status === 'failed' && (
               <Alert kind="danger" children={attempt.statusText} />
             )}
-            <FieldInput
-              label="Username"
-              rule={requiredField('Username is required')}
-              placeholder="Username"
-              autoFocus
-              value={name}
-              onChange={e => onChangeName(e.target.value)}
-              readonly={isNew ? false : true}
-            />
-            <FieldSelectAsync
-              menuPosition="fixed"
-              label="User Roles"
-              rule={requiredField('At least one role is required')}
-              placeholder="Click to select roles"
-              isSearchable
-              isMulti
-              isSimpleValue
-              isClearable={false}
-              value={selectedRoles}
-              onChange={values => onChangeRoles(values as Option[])}
-              noOptionsMessage={() => 'No roles found'}
-              loadOptions={async input => {
-                const roles = await fetchRoles(input);
-                return roles.map(r => ({ value: r, label: r }));
-              }}
-              elevated={true}
-            />
+            <Box maxWidth={690}>
```

Comment:
is this maxWidth required? (we already set a maxWidth on the parent)

Reply:
This is required for inner content (added in https://github.com/gravitational/teleport/pull/42620/commits/7d4ecd8ed7807ce61c04b1ea64dafbce77eb2903). Without this, the inner content shrinks when vertical scrollbar appears, causing flickering.